src/utils/detector.py

```diff
@@ -0,0 +1,6 @@
+def identify(text):
+    keywords = ['meeting', 'party']
+    for keyword in keywords:
+        if keyword in text:
```

Comment:
It's not necessary we use word meeting and party other words may also be there. So make sure you have some context driven model here for identifying the context of the messages and then classify it as a meeting message or party etc. depends on the type of categories you are considering.

Reply:
Yes, I will look into this . I wanted to make a model but didn't know how to create training data.

Reply:
Do the same so that I can merge it.

Reply:
You will be having some messages as your training data.